README.md

```diff
@@ -481,3 +481,12 @@ onExit() {
   openPage(router, 'home')
 }
 ```
+
+If you need to replace current URL programmatically you can use `replacePage`:
+
+```ts
+onExit() {
```

Comment:
It is not cool that we use the same example in a different use cases. It is better to show a good use case for `replacePage` or change `openPage` example (if `replacePage` is the best for exit).

Reply:
fixed